### ISSUE TITLE
[FEAT] Bypass, skip or set own my thresholds for specific values and/or attributes

### ISSUE THREAD

**Issue by Smexhy** [AnalogJ/scrutiny#369]
**Is your feature request related to a problem? Please describe.**
Some time ago I was opening issue regarding certain value and threshold which was resulting in my checks as failed for this specific disk, because of Command Timeout attribute. https://github.com/AnalogJ/scrutiny/issues/260 - I noticed that daevin commented on this issue as well after it was closed few months ago, but I couldn't verify his statement personally, what do you think? Although I am opening this issue from another reason. 

**Describe the solution you'd like**
I would like to be able to set threshold for specific attributes myself, or completely bypass checks for them for certain drives. Currently I can set in webUI settings under Device Status - Thresholds to only use SMART thresholds, not scrutiny (or both), and that actually fixed my issue and shows the drive checks as passed. However when I open drive additional info to see specific thresholds one by one, the command timeout attribute still shows as failed, even with my settings set to SMART only. I would like to either so that scrutiny would use SMART thresholds after opening additional drive info as well, or to be able to set manual scrutiny thresholds for specific attributes, so for example if I could set threshold of Command Timeout to 2 for my ATA drive and therefore making scrutiny validate disk as "passed".

**Comment by dmshimself** [comment]
Oh yes please.  In particular being able to control when Command Timeout value changes kick in.

**Comment by rophel** [comment]
Would love this for Command Timeout.

I believe I had a bad cable that caused three drives to have very high Command Timeout numbers.  Nothing else is wrong with any of the drives besides that value.  I had rotated them around in my chassis attempting to diagnose and all three used that cable until I identified the issue and replaced it.  The number has not gone up since.

**Comment by yorah** [comment]
Same thing happened to me: a bad cable made my command timeout attribute jump through the roof. Too bad that there is no way to "reset" (I understand SMART is not supposed to be resettable), but if at least we could specify our own thresholds to take those cases into account, that would be great.

**Comment by CannyGent** [comment]
Yes - i need this feature too... i think my Command Timeout values are all high as have had drives in USB-C enclosures and/or set to spindown.

**Comment by pax0707** [comment]
Just had this happen...
+1

**Comment by kaysond** [comment]
This issue is being closed automatically as a part of an effort to streamline and prioritize development effort. If it is still relevant, please reply to this comment and we will convert it to a discussion.
